tensorboard/webapp/widgets/card_fob/card_fob_types.ts

```diff
@@ -29,35 +29,24 @@ export enum AxisDirection {
 }
 
 /**
- * This class is intended to be implemented by the card that has a
+ * These helper functions are intended to be used in the card that has a
```

Comment:
"These helper functions are intended to be defined by the card that is a parent of CardFobControllerComponent." ?